src/cartridge_header.rs

```diff
@@ -1,62 +0,0 @@
-pub(crate) struct CartridgeHeader {
```

Comment:
You completely deleted a initial implementation of Cartridge Header. Can you merge this file with your new implementation?

Reply:
I just moved it in the new file

Reply:
Nevermind, I realized now that the implementation was changed, maybe it was added when I rebased the branch. I'll bring changes to the new file then